src/theme.scss

```diff
@@ -1,10 +1,11 @@
 // Default theme. Designed to be used as a starting point for your custom theme.
 
-@use "styles/theme-constants/breakpoints";
-@use "styles/theme-constants/colors";
-@use "styles/theme-constants/svg";
+@use "src/styles/theme-constants/breakpoints";
+@use "src/styles/theme-constants/colors";
+@use "src/styles/theme-constants/svg";
```

Comment:
Why `src` in the path? Is it somehow required for generating the distribution files?

Reply:
So I had to return them, it does not work otherwise…

Reply:
@adamkudrna will take another look

Reply:
We tested it and foud out `src/` is not necessary in any paths.